audobject/core/resolver.py

```diff
@@ -179,6 +179,25 @@ class Function(Base):
     Encodes source code of function and
     dynamically evaluates it when the value is decoded again.
 
+    Note that a decoded function
+    might raise a :class:`NameError`,
+    if it relies on other locally defined functions.
```

Comment:
Should we also explicitly mention that it must not rely on functions that are imported outside the function?

Reply:
Good point, forgot about that.

Changed now to the more general text:

```
    Note that a decoded function
    might raise a :class:`NameError`,
    if it relies on other objects,
    that are locally defined,
    or imported outside of the function.
```